src/containers/Home/index.js

```diff
@@ -1,86 +1,76 @@
 //@flow
-import React, { Component } from 'react';
-import { View } from 'react-native';
-import styles from './styles';
-import Header from 'src/components/Header'
-import CommonHelper from 'src/utils/helper';
-import GeoMap from 'src/components/GeoMap';
-import Button from 'src/components/Button';
-import type { Region } from 'src/types/GeoMapTypes';
+import React, { Component, useEffect, useState } from "react";
+import { View } from "react-native";
+import styles from "./styles";
```

Comment:
should be imported last. all css files should be imported at the end of import list